images/macos/toolsets/toolset-12.json

```diff
@@ -15,19 +15,26 @@
             "versions": [ "2019", "2022" ]
         },
         "mono-versions": [
-            "6.12.0.174"
+            "6.12.0.179"
```

Comment:
we need to install 6.12.0.182 using different link for now https://download.mono-project.com/archive/6.12.0/macos-10-universal/MonoFramework-MDK-6.12.0.182.macos10.xamarin.universal.pkg

Reply:
Added the new version of mono